### PR TITLE
added new rules

### PR DESCRIPTION
**Pull request by hrshdhgd**
- methylase activity => methylation, 
 - polymerase activity=>polymerization [*ase activity->ation]

### CODE REVIEW

**Comment by hrshdhgd** [comment]
Once approved, I'll run the term list through ontoRunNER and post results.

File: rulebook/synonym_rules.yaml [hunk 1]
```diff
@@ -61,6 +61,28 @@ rules:
...
+    replacement: "\\1ation"
+    replacement_scope: exact
+    tests:
+      - input: methylase activity
+        output: methylation
+        ontology: GO
+  - type: NewSynonym
+    branches:
+      - biological_process
+    match: "(.*)rase activity"
```
Comment (cmungall) [inline rulebook/synonym_rules.yaml:78]: this is a bit too general - e.g transferase->transferization doesn't make sense

lt's hardcode polymerase->polymerization for now

Comment (cmungall) [reply rulebook/synonym_rules.yaml:78]: oh i see we have that below...

Comment (cmungall) [reply rulebook/synonym_rules.yaml:78]: ok, I think this is good, we have to expect a few nonsensical generated synonyms, it's oK as they won't actually be used in text

**Review by cmungall — APPROVED** [review]

**Pull request merged**